assets/assetpack.def

```diff
@@ -57,6 +58,7 @@
 < https://raw.githubusercontent.com/twbs/bootstrap-sass/5d6b2ebba0c2a5885ce2f0e01e9218db3d3b5e47/assets/javascripts/bootstrap/tab.js
 < https://raw.githubusercontent.com/twbs/bootstrap-sass/5d6b2ebba0c2a5885ce2f0e01e9218db3d3b5e47/assets/javascripts/bootstrap.js
 < https://raw.githubusercontent.com/twbs/bootstrap-sass/5d6b2ebba0c2a5885ce2f0e01e9218db3d3b5e47/assets/javascripts/bootstrap/transition.js
+< https://github.com/sorich87/bootstrap-tour/blob/v0.11.0/build/js/bootstrap-tour.min.js
```

Comment:
do not add .min but add the real thing - and the URL you add is not a download link but a page showing it, i.e. is HTML